config/locales/en.yml

```diff
@@ -312,30 +312,45 @@ en:
       - title: Working, jobs and&nbsp;pensions
         description: Includes holidays, finding a job and redundancy
         link: /browse/working
+    government_activity:
+      - title: News
```

Comment:
Because we're doing some explicit column setting, I'd advise adding a comment reminding folks that if they add something here, they need to updated the associated css. [Here's an example on my more on govuk PR](https://github.com/alphagov/frontend/pull/3039/files#diff-44438ce218f5287c58d0017f965d888715635d94280669896f75841fbd7b4cd7R363-R364).